nix/shell-plugins.nix

```diff
@@ -0,0 +1,57 @@
+{ pkgs, lib, config, is-home-manager, ... }:
+with lib;
+let cfg = config.programs._1password-shell-plugins;
+in {
+  options = {
+    programs._1password-shell-plugins = {
+      enable = mkEnableOption "1Password Shell Plugins";
```

Comment:
`mkEnableOption` is a shorthand to just create a `boolean` option with description `"Whether to enable ${opt}"`, where `opt` is the text we're passing here ("1Password Shell Plugins").